types/sinon/index.d.ts

```diff
@@ -322,8 +318,15 @@ declare namespace Sinon {
         restore(): void;
     }
 
+    interface SinonFakeServerOptions {
+        autoRespond: boolean;
```

Comment:
Should these actually be each individually optional too? I guess you don't need to provide all options when you pass in a config object.

Reply:
ah you have a very good point. im having some build issues locally so somehow didn't notice this!